components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -58,6 +58,7 @@ public static class ExitCode {
 
     @Override
     public Integer call() throws Exception {
+        List<Build> builds = buildsFrom(runNumsAndBuildScanUrls);
```

Comment:
new line after

Reply:
or move to right above the `customValueKeys` declaration

Reply:
✅ Done